Tools/AP_Periph/Parameters.cpp

```diff
@@ -224,6 +228,13 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @Group: BATT
     // @Path: ../libraries/AP_BattMonitor/AP_BattMonitor.cpp
     GOBJECT(battery_lib, "BATT", AP_BattMonitor),
+
+    // @Param: BATT_HIDE_MASK
+    // @DisplayName: Battery hide mask
+    // @Description: Instance mask of battery index(es) to not transmit their status over CAN. This is useful for hiding a "battery" instance that is used by other libraries but don't want to be treated as a battery source to the autopilot. For example, an AP_Periph based motor controller (ESC) that is reporting as an ESC. It should not be reporting "batteries", only ESC telemetry type data which includes voltage and current.
```

Comment:
Tom, the example is unclear to me..is it to mask the battery  data being provided by an ESC's telem but you want to ignore it? to use the data only locally in AP_Periph? the parm is only internal to an AP_Periph device right?

Reply:
>also can a masked periph batt mon instance be used by the autopilot for it own peripheral batt monitor
I don't understand. A Periph is external hardware from the autopilot. This is a mask to hide a battery in a periph so the autopilot wouldn't know of it, only the periph. Yes, to be used locally. Randy did not like my use of the word "internal" so I was trying to avoid using that word.

Reply:
if you mask a batt monitor in the periph (the param is a periph param, not an autopilot one, right?) can the autopilot assign an identical monitor instance to another attached monitor

Reply:
they're totally unrelated

Reply:
I'm open to suggestions